Prisma/Standard/PrismaStandard+HealthKit.swift

```diff
@@ -80,6 +80,7 @@ extension PrismaStandard {
             // timeIndex is a dictionary with time-related info (range, timezone, datetime.start, datetime.end)
             // timeIndex is added a field for this specific HK datapoint so we can just access this part to fetch/sort by time
             firestoreResource["time"] = timeIndex
+            firestoreResource["datetimeStart"] = timeIndex
```

Comment:
This doesn't seem right to me. Why are you assigning the timeIndex again?

Reply:
Just fixed this and also set order to be in descending order.